x-pack/auditbeat/processors/sessionmd/add_session_metadata.go

```diff
@@ -35,10 +36,13 @@ func InitializeModule() {
 }
 
 type addSessionMetadata struct {
+	ctx      context.Context
+	cancel   context.CancelFunc
 	config   config
 	logger   *logp.Logger
 	db       *processdb.DB
 	provider provider.Provider
+	backend  string
 }
 
 func New(cfg *cfg.C) (beat.Processor, error) {
```

Comment:
Will you please add a cfgwarn statement because this is beta. e.g.

`cfgwarn.Beta("add_session_metadata processor is a beta feature.")`